src/test/java/com/google/cloud/genomics/utils/grpc/TestHelper.java

```diff
@@ -60,6 +60,38 @@
         .addAllGenotype(Arrays.asList(gt));
   }
 
+  public static Variant.Builder makeVariant(String chr, long start, String ref, String... alts) {
+    return Variant.newBuilder()
+        .addFilter("PASS")
+        .setReferenceName(chr)
+        .setStart(start)
+        .setEnd(start + ref.length())
+        .setReferenceBases(ref)
+        .addAllAlternateBases(Arrays.asList(alts));
+  }
+
+  public static Variant.Builder makeVariant(
+      String chr, String id, String ref, String alt, List<VariantCall> calls) {
+    return Variant.newBuilder()
+        .addFilter("PASS")
+        .setReferenceName(chr)
+        .setId(id)
+        .setReferenceBases(ref)
+        .addAlternateBases(alt)
+        .addAllCalls(calls);
+  }
+
+  public static Variant.Builder makeVariant(String chr, long start, String ref, String alt, double quality) {
```

Comment:
Could these three new test helpers be combined into one?  This file was already a bit long and now its harder to differentiate which makeVariant to use.

```
public static Variant.Builder makeVariant(String chr, String id, long start,
                                                                  String ref, List<String> alts,
                                                                  String filter, double quality, List<VariantCall> calls) {
```